AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoConstants.h

```diff
@@ -314,3 +314,20 @@ extern NSString * const kAlfrescoWorkflowVariableTaskInvitePendingOutcome;
 extern NSString * const kAlfrescoWorkflowTaskTransitionApprove;
 extern NSString * const kAlfrescoWorkflowTaskTransitionAccept;
 extern NSString * const kAlfrescoWorkflowTaskTransitionReject;
+
+/**---------------------------------------------------------------------------------------
+ * @name Connection Diagnostic Constants
+ --------------------------------------------------------------------------------------- */
+extern NSString * const kAlfrescoConfigurationDiagnosticDidStartEventNotification;
+extern NSString * const kAlfrescoConfigurationDiagnosticDidEndEventNotification;
+
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsLoading;
+extern NSString * const kAlfrescoConfigurationDiagnosticDictionaryIsSuccess;
```

Comment:
isSuccess can hold both YES and NO, so to make it into only one key then the value would have to be a number created from an enum that would have (initially) 3 states: loading, success, failed.